openprescribing/measure_definitions/hospital_only.json

```diff
@@ -14,8 +14,8 @@
     " you should contact the NHS Business Services Authority who maintain the dm+d at <a href='mailto:nhsbsa.dmdenquiries@nhs.net?Subject=hospital%20only'>nhsbsa.dmdenquiries@nhs.net</a>.",
     "This measure suports review and audit of appropriateness as sometimes a hosptal only",
     "medicine may be selected in error for example <a href='https://ebmdatalab.net/zuclopenthixol-acetate-a-new-kind-of-measure-on-openprescribing/'> zuclopenthixol acetate (blog here)</a>.</p>",
-    "<p>Please note that this is an experimental measure. We would be grateful for",
-    "any feedback at <a href='mailto:feedback@openprescribing.net?Subject=hospital%20only'>feedback@openprescribing.net</a></p>"
+    "<p>Please note that this is an experimental measure and we have written more about it on <a href='https://ebmdatalab.net/nhs-dmd-hospital-only/'> our blog here/a>. We",
```

Comment:
`/a>` should be `</a>`

Reply:
I have just made this change - tests running.

How did you spot this? Was it by eye or do you have software that checks?

Reply:
By eye.  We really should have an automatic test to catch this kind of thing though.  See #2729.